src/SimpleWallet/SimpleWallet.cpp

```diff
@@ -372,6 +399,154 @@ struct TransferCommand {
 
     return true;
   }
+
+  bool parseMessageArgs(LoggerRef& logger, const std::vector<std::string>& args) {
```

Comment:
Inside this function each tab character must be replaced with 4 spaces. Please, never use tabs for indentation!